### DISCUSSION TITLE
How to change the default editor for opening the file?

### DISCUSSION THREAD

**Discussion by abhiche** [sayanarijit/xplr#538] [Q&A]
After selecting a file, when I press `:e`, the file gets opened in vim. 
I am currently using helix editor.
How to open the file in helix?

**Comment by sayanarijit** [comment]
Add `export EDITOR=hx` in your .zshrc/.bashrc/.profile.

Or override [this key binding](https://github.com/sayanarijit/xplr/blob/25798009adb2145f5aa904dfe0e65c3518aeed3a/src/init.lua#L1857) in your custom init.lua.

**Reply by abhiche** [reply]
I am getting the below error now when trying to open a file.
```
ERROR: process exited with code exit status: 101
```

```
~ ENV | grep EDITOR
EDITOR=hx
~ hx --version
helix 22.08.1 (66276ce6)
~ xplr --version
xplr 0.20.0
```

**Reply by sayanarijit** [reply]
If you're using mac, I believe you're running into https://github.com/sayanarijit/xplr/issues/449.

The current solution is to use a [custom build of helix](https://github.com/helix-editor/helix/issues/2111).

**Reply by abhiche** [reply]
Thanks. Looks like it.
I am enjoying using xplr. You are doing an amazing job. 🙇

**Reply by sayanarijit** [reply]
Thank you, and thanks to the opensource community 🙂
